qsc/tests/test_grad_B_tensor.py

```diff
@@ -121,5 +121,63 @@ def test_axisymmetry(self):
                         np.testing.assert_allclose(s.grad_grad_B_alt[:,0,2,0], np.full(nphi, val), rtol=rtol, atol=atol)
                         np.testing.assert_allclose(s.grad_grad_B_alt[:,0,0,2], np.full(nphi, val), rtol=rtol, atol=atol)
                         
+
+class MagneticFieldTests(unittest.TestCase):
+
+    def test_magnetic_field_nabla(self):
+        """
+        Test the magnetic field vector and its derivatives
```

Comment:
How about adjusting this wording to better reflect what is being tested here: these tests are specifically about the cylindrical and Cartesian versions of the grad B and grad grad B tensors.